### PR TITLE
adding _redirects and gatsby-plugin-netlify

### LINKED ISSUE #70: Fix broken links and search results
This is a single issue for tracking broken links and search results that end up nowhere.

## broken links
Also mentioned:
- https://github.com/operate-first/community/issues/215

Ideas/ Problems:
- link issues can go in different repos (see mentioned above), e.g., support, community, and not necessarily here
  - how to catch them better
- link checker as a service
  - 1. gatsby plugin? https://www.gatsbyjs.com/plugins/gatsby-remark-check-links/
  - 2. "cron job"?
  - 3. manual regularly https://github.com/linkchecker/linkchecker

## search results
how can be correct search results? sitemap?

### LINKED ISSUE #200: replace operate-first.cloud/data-science/operate-first-data-science-community/docs/content.md
operate-first.cloud/data-science/operate-first-data-science-community/docs/content.md
to
old.operate-first.cloud/data-science/operate-first-data-science-community/docs/content.md

also check other operate-first.cloud/data-science/operate-first-data-science-community/ links

### PR DESCRIPTION
**Pull request by schwesig**

### CODE REVIEW

**Comment by schwesig** [comment]
Redirects tested on the Deploy Preview above.
These are working.

problem: I couldn't find a way to not pointing to asingle page; so every page needs an entry.
preferred: redirect is using only the path and add the single pages/ details from the FROM
e.g. 
instead of
``` /data-science/operate-first-data-science-community/docs/content.md https://old.operate-first.cloud/data-science/operate-first-data-science-community/docs/content.md```
something like
``` /data-science/operate-first-data-science-community/docs/* https://old.operate-first.cloud/data-science/operate-first-data-science-community/docs/*```

I tried
```/api/*  https://api.example.com/:splat  200```
from
https://docs.netlify.com/routing/redirects/rewrites-proxies/#shadowing
but I couldn't get it to work.

Other resources I tried:
https://levelup.gitconnected.com/how-to-add-netlify-redirects-to-a-gatsby-site-ae678518cc91
https://www.gatsbyjs.com/plugins/gatsby-plugin-netlify/
https://play.netlify.com/redirects
https://www.gatsbyjs.com/docs/reference/routing/file-system-route-api/#splat-routes

**Comment by schwesig** [comment]
finally found the blocker:
pre-commit changed the (correct)
```/data-science/* https://old.operate-first.cloud/data-science/:splat 301!```
to (the not working)
```/data-science/\* https://old.operate-first.cloud/data-science/:splat 301!```
or
```/data-science/_ https://old.operate-first.cloud/data-science/:splat 301!```

now, paths and :splat is working.

/remove do-not-merge

**Comment by schwesig** [comment]
/fixes https://github.com/operate-first/operate-first.github.io/issues/200

**Comment by schwesig** [comment]
fixes https://github.com/operate-first/operate-first.github.io/issues/200

**Comment by schwesig** [comment]
this fixes links like
operate-first.cloud/community-handbook/...
operate-first.cloud/data-science/...

therefore
fixes https://github.com/operate-first/operate-first.github.io/issues/70
fixes https://github.com/operate-first/community/issues/215

File: static/_redirects [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# manual redirects
```
Comment (codificat) [inline static/_redirects:1]: Ok, so this uses [netlify redirects](https://docs.netlify.com/routing/redirects/#syntax-for-the-redirects-file), right? (and therefore it works in the netlify-provided preview)

However, the website is served from GitHub pages, not netflity, rigth? Does that work with GitHub pages?

Comment (durandom) [reply static/_redirects:1]: Looking at https://docs.netlify.com/routing/overview/

> As requests come in from site visitors, our CDN edge nodes evaluate these rules, pointing visitors to different paths or rewriting paths based on options or conditions like preferred language or current location.

I don't think this work. @schwesig does it work locally?

**Pull request closed without merging**